newsroom/push.py

```diff
@@ -413,6 +413,14 @@ def should_add(date):
     return display_dates
 
 
+def set_coverage_delivery_href(coverage, slugline):
```

Comment:
I think we will need some api to define the function for delivery href rather than using just photo url config and adding some hardcoded params, maybe defining `app.delivery_href` function which be noop by default and could be set in some app specific delivery module

Reply:
Cool, I wasn't sure if this was the right way. Discussde with @mdhaman I'll change the PR to have an API and noop